ls1intum/Artemis
# `Programming exercises`: Remove mention of SourceTree for cloning

### Checklist
#### General


#### Client

### Motivation and Context


The tutorial is currently switched from SourceTree to only using IntelliJ for cloning (#5988), so the clone dialog should not put such a big emphasis on using source tree, let alone make an advertisement for this specific git client.

### Description

This PR makes the button label more generic and adds a small help icon, explaining the buttons usage.
The direct link to SourceTree below the button is removed.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Programming Exercise

1. Log in to Artemis
2. Navigate to Course Administration
3. Try to clone the exercise using the "Clone in git client" button in these cases:
- Without any git client installed (button should do nothing)
- With either SourceTree or Tower installed (button should open the respective git client)(Not every tester has to check every git client)

### Review Progress






#### Code Review
#### Manual Tests

### Screenshots


![Bildschirm­foto 2023-01-13 um 16 59 49](https://user-images.githubusercontent.com/38322605/212364198-4bf62be7-b93c-4533-a5a8-706ee99dfbb6.png)
